pkg/api/rpms.go

```diff
@@ -26,6 +26,31 @@ type RepositoryRpmCollectionResponse struct {
 	Links Links            `json:"links"` // Links to other pages of results
 }
 
+type SnapshotErrataListRequest struct {
+	UUID     string   `param:"uuid"`     // Identifier of the repository
+	Search   string   `query:"search"`   // Errata Id to optionally filter-on
+	Type     []string `query:"type"`     // Type to optionally filter-on
+	Severity []string `query:"severity"` // Severity to optionally filter-on
+}
+
+type SnapshotErrata struct {
+	Id              string `json:"id"`
+	ErrataId        string `json:"errata_id"`        // Id of the errata
```

Comment:
same question as above
```suggestion
	ErrataId        string `json:"errata_id"`        // ID of the errata
```